notebooks/nuextract-structure-extraction/nuextract-structure-extraction.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from IPython.display import display, Markdown
</pre><p>I do not think that somebody needs all 3 variants of models in the same time, from UX perspective it is enough to select precision one time for both conversion and usage. I also recommended to reuse conversion code and widgets from llm_config.py</p><p><a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/llm_config.py#L495-L610" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/llm_config.py#L495-L610</a></p><p> </p><p>please check llm-chatbot-genai notebook for details</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Fixed</p>